### ISSUE TITLE
Basic example doesnt work

### ISSUE THREAD

**Issue by staabm** [nanasess/setup-php#13]
Based on the basic example I tried to setup the action in
https://github.com/staabm/SysMonitor/pull/16/files

I am running into the following error
```

Current runner version: '2.158.0'
2
Prepare workflow directory
3
Prepare all required actions
4
Download action repository 'actions/checkout@master'
5
Download action repository 'nanasess/setup-php@v2'
6
##[warning]Failed to download action 'https://api.github.com/repos/nanasess/setup-php/tarball/v2'. Error Response status code does not indicate success: 404 (Not Found).
7
##[warning]Back off 23.963 seconds before retry.
8
##[warning]Failed to download action 'https://api.github.com/repos/nanasess/setup-php/tarball/v2'. Error Response status code does not indicate success: 404 (Not Found).
9
##[warning]Back off 14.855 seconds before retry.
```
https://github.com/staabm/SysMonitor/pull/16/checks?check_run_id=231534011

**Comment by staabm** [comment]
With
  uses: nanasess/setup-php@master

Instead of
  uses: nanasess/setup-php@v2

It seems to work now.

Running the actions seems to be pretty slow so far
![1222633A-1EDF-4574-BE45-C7CF20000A48](https://user-images.githubusercontent.com/120441/65386542-5e848f00-dd3d-11e9-9cd3-3feb3a674b85.jpeg)

**Comment by nanasess** [comment]
@staabm 
Fix version in README(#15)
Thanks for the feedback.

**Comment by staabm** [comment]
thx
